src/Shopsys/ShopBundle/Resources/views/Front/Content/Product/filterFormMacro.html.twig

```diff
@@ -43,36 +43,6 @@
             </label>
         </div>
 
-        {% if filterForm.flags is empty %}
-            {% do filterForm.flags.setRendered %}
```

Comment:
I believe this must be kept, otherwise the checkboxes are still rendered by default in the parametric filter (I tested this :slightly_smiling_face: )

Reply:
In the end, I removed it. But I changed ProductFilterFormType in which the flags remained. Now it should be fine :)